https-upgrades/README.md

```diff
@@ -77,5 +77,6 @@ thirdnegative.com
 
 ## Platform exceptions
 
-- Android, iOS and macOS don't support negative bloom filter because it's only needed for platforms where Smarter Encryption API is available
+- Android, Windows, iOS and macOS don't support negative bloom filter because it's only needed for platforms where Smarter Encryption API is available
 - Android, iOS and macOS don't support updating subrequests because of WebView limitations (request can be cancelled, but not modified)
+- Windows don't support updating websocket subrequests because of WebView limitations (request can be cancelled, but not modified)
```

Comment:
Hey @szanto90balazs ! Could you please link this limitation to an asana task? We should remember to remove this once the limitation goes away.

Reply:
@RendijsSmukulis Do you know if we have one?